docs/editor_manual/browser_issues.rst

```diff
@@ -0,0 +1,11 @@
+Browser issues
+~~~~~~~~~~~~~~
+
+Some issues with Wagtail may come from the browser. Try changing or resetting its settings, or `clearing the browser cache and cookies <http://www.refreshyourcache.com/>`_. You can also try using Wagtail with `another browser <http://browsehappy.com/>`_ to see if the problem persists.
```

Comment:
Good point. I think it will be hard to suggest specific actions (apart from having JS enabled, which is done below) so I'll remove this.